src/api/options-assets.md

```diff
@@ -8,6 +8,22 @@
 
   A hash of directives to be made available to the Vue instance.
 
+- **Usage:**
+  ```js
+  const App = Vue.createApp()
```

Comment:
**suggestion**: fixed a wrong capital letter on name and added param passed to `createApp` (unfortunately we would have a warning if nothing is passed to `createApp` even if it's an empty object

```suggestion
  const app = Vue.createApp({})
```